kuma/wiki/jinja2/wiki/list/documents.html

```diff
@@ -12,6 +12,7 @@
   {% set title = _('All documents') %}
 {% endif %}
 {% block title %}{{ page_title(title) }}{% endblock %}
+{% block robots_value %}noindex, nofollow{% endblock %}
```

Comment:
I think we want list pages followed. That's [one of these pages](https://developer.mozilla.org/en-US/docs/tag/HTML), right?